os/fs/procfs/fs_procfsproc.c

```diff
@@ -953,6 +979,52 @@ static ssize_t proc_entry_groupfd(FAR struct proc_file_s *procfile, FAR struct t
 	return totalsize;
 }
 
+#if defined(CONFIG_PM) && defined(CONFIG_PM_METRICS)
+static ssize_t proc_entry_pminfo(FAR struct proc_file_s *procfile, FAR struct tcb_s *tcb, FAR char *buffer, size_t buflen, off_t offset)
```

Comment:
fs_procfsproc.c is to handle info regarding pid, if pm working related to each of pid then this can be acceptable, otherwise, please make an extra file to handle this.